<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflectanceModelDockWidgetBase</class>
 <widget class="QDockWidget" name="ReflectanceModelDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reflectance model</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="reflectanceModelComboBox"/>
    </item>
    <item>
     <widget class="QWidget" name="parameterWidget" native="true">
      <layout class="QFormLayout" name="parameterFormLayout"/>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Coordinate System</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="coordSysComboBox">
      <item>
       <property name="text">
        <string>Half difference</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Specular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spherical</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Division number</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="coordSysStackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="halfDiffCoordSysPage">
          <layout class="QFormLayout" name="halfDiffCoordSysFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Half polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="halfDiffCsNumAngle0SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="halfDiffCsNumAngle1Label">
             <property name="text">
              <string>Half azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="halfDiffCsNumAngle1SpinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>72</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Difference polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="halfDiffCsNumAngle2SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>36</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Difference azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="halfDiffCsNumAngle3SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>18</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="specularCoordSysPage">
          <layout class="QFormLayout" name="specularCoordSysFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Incoming polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="specularCsNumAngle0SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>18</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Specular polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="specularCsNumAngle2SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Specular azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="specularCsNumAngle3SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>72</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="specularCsNumAngle1Label">
             <property name="text">
              <string>Incoming azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="specularCsNumAngle1SpinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>36</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="sphericalCoordSysPage">
          <layout class="QFormLayout" name="sphericalCoordSysFormLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::FieldsStayAtSizeHint</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Incoming polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sphericalCsNumAngle0SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>18</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Outgoing polar angle:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sphericalCsNumAngle2SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>90</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Outgoing azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sphericalCsNumAngle3SpinBox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>72</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="sphericalCsNumAngle1Label">
             <property name="text">
              <string>Incoming azimuthal angle:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sphericalCsNumAngle1SpinBox">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>36</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="generateBrdfPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Generate BRDF</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>reflectanceModelComboBox</tabstop>
  <tabstop>coordSysComboBox</tabstop>
  <tabstop>halfDiffCsNumAngle0SpinBox</tabstop>
  <tabstop>halfDiffCsNumAngle1SpinBox</tabstop>
  <tabstop>halfDiffCsNumAngle2SpinBox</tabstop>
  <tabstop>halfDiffCsNumAngle3SpinBox</tabstop>
  <tabstop>specularCsNumAngle0SpinBox</tabstop>
  <tabstop>specularCsNumAngle1SpinBox</tabstop>
  <tabstop>specularCsNumAngle2SpinBox</tabstop>
  <tabstop>specularCsNumAngle3SpinBox</tabstop>
  <tabstop>sphericalCsNumAngle0SpinBox</tabstop>
  <tabstop>sphericalCsNumAngle1SpinBox</tabstop>
  <tabstop>sphericalCsNumAngle2SpinBox</tabstop>
  <tabstop>sphericalCsNumAngle3SpinBox</tabstop>
  <tabstop>generateBrdfPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>coordSysComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>coordSysStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
